static/js/env.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * Copyright (c) 2014-2015, CKSource - Frederico Knabben. All rights reserved.
+ * Licensed under the terms of the MIT License (see LICENSE.md).
+ */
+
+( function() {
+	var env = {},
+		ua = navigator.userAgent;
+
+	env.supportsConsole = !!( window.console && window.console.log );
+
+	env.trident = ua.toLowerCase().indexOf( 'trident' ) > -1;
+
+	env.spartan = ( /edge[ \/]\d+.?\d*/ ).test( ua.toLowerCase() );
+
+	env.ie = env.trident || env.spartan;
```

Comment:
It could be a good idea. https://d10.github.io/blog/platform.js-v1.3.0/ or http://www.uaparser.org/ if they are available on npm.